contracts/release1/build2/build-metadata-R1B2.json

```diff
@@ -0,0 +1,37 @@
+{
+  "ui": "",
+  "change": "Added `address account` as a second state variable.",
+  "pluginSetupABI": {
```

Comment:
Wondering if there's a better name for this
- `pluginSetup`
- `lifecycleABI`
- `lifecycleParams`
- ...

Reply:
The functions 
`prepareInstallation`, `prepareUpdate`, `prepareUninstallation` that require the arguments that we describe with types and names are located in the `PluginSetup` contract.
Thus, naming this the `pluginSetup` is good, and I would not refrain from that. 

Since what we write there is not exactly the ABI (the contract ABI is a standardized way of encoding its functions and parameters into a format that can be understood by other programs or contracts) I have nothing against dropping the `ABI` part from the name.

In this context, talking about lifecyles makes things more vague IMO.

WDYT @Rekard0 ?

Reply:
Changed in aff41b9b35e41c3b5745e9e8d3f820637112a858